server/utils/accessControl.js

```diff
@@ -36,8 +36,10 @@ const checkAdminAccess = (user) => {
 // WARNING: the param name needs to be exactly 'countryIso'
 // If it's not, use checkCountryAccess instead
 const checkCountryAccessFromReqParams = (req) => {
-  if (req.params.countryIso) checkCountryAccess(req.params.countryIso, req.user)
-  if (req.query.countryIso) checkCountryAccess(req.query.countryIso, req.user)
+  if (req.user) {
```

Comment:
is this ```if(req.user``` still needed ?

Reply:
I need to figure a better way - the underlying methods are using the user and failing if the user doesn't exist I think ( user = 'undefined' )

Reply:
Right, but this method should be invoked only if the user exists. Isn't it so?